site/student-lectures/Evolutionary Art/Evolutionary Art.rst

```diff
@@ -0,0 +1,160 @@
+****************
+Evolutionary Art
+****************
+
+* Evolutionary Art represents the convergence of Creative Expression and Evolutionary Computing
+* The evolutionary period often produces new forms of art
+
+.. figure:: apespeare.jpg
+    :width: 500 px
+    :align: center
+
+    A Banana by any Other Name
+
+Evolutionary Artforms
+=====================
+
+* This is really just "How do humans make art?"
+
+Music & Dance
+-------------
+* Make melodies and full songs
+* Develop rhythms and musical structure
+* Create sequences of choreography
+
+Paintings & Sculpting
+---------------------
+* Attempts to achieve a target solution (TSP? Or SymboReg)
+
+.. figure:: polylisa.webp
+    :width: 500 px
+    :align: center
+
+    Poly Lisa
+
+* Complete Randomization of art
+* Creating 3d Sculptures that can be modeled
+
+Poetry & Shakespeare (or really… any book)
+------------------------------------------
+* Attempts to form grammatically correct sentences or follow rules of poetry
+
+.. note::
+    Follow the rules of a Haiku: 
+        - Start and end with five,
+        - Middle line houses seven, 
+        - Structured syllables
+
+Actually, I think math is art 🤓
+--------------------------------
+* Okay
+
+* See Assignment Two
+
+Making an Evolutionary Program
+==============================
+* What is a population?
+* How could you possibly give a fitness score? Isn’t art subjective?
+* What does crossover look like? 
+* Mutation?
+* Elites? Like Van Gogh? 
+
+.. figure:: PaintingCrossover.png
+    :width: 500 px
+    :align: center
+
+    Painting Crossover Example?
+
+Encoding Art As Chromosomes
+---------------------------
+
+* Visual Art
+    * Create nxm pixel lists that can have any colour in any give spot
+    * Mathematical functions that make specific shapes
+
+.. figure:: desmoslove.webp
+    :width: 500 px
+    :align: center
+
+    Mathematical Love 
+
+* Music
+    * Store Musical Components as Variables (Like chords)
+    * MIDI representations that can be stored as numeric values
+
+.. figure:: midiart.png
+    :width: 500 px
+    :align: center
+
+    Music to MIDI
+
+.. note::
+    For example, Hot cross buns follows: 
```

Comment:
if you add a new line between the top like and the points, the top line wont be boldfaced.

![image](https://github.com/user-attachments/assets/644a19d0-312f-486f-b85a-8d2b74a0920d)

It doesn't really matter, but I doubt this was intended.

Reply:
Not intended, but I kind of like it. Thanks for looking at it so quickly.